.travis.yml

```diff
@@ -25,6 +25,7 @@ before_install:
   - python -m pip uninstall -y pyccel
   - python -m pip uninstall -y gelato
   - python -m pip install -r requirements.txt
+  - python -m pip install numba
```

Comment:
While Pyccel is a hard dependency for Psydac, Numba is not. Instead, Numba is just a backend for the acceleration of our generated Python code. 

Therefore I think we should install Numba **after** Psydac, just below line 32. (This is what we do in Pyccel: see [here](https://github.com/pyccel/pyccel/blob/master/.travis.yml).)

Reply:
done